pkg/api/api.go

```diff
@@ -258,7 +258,8 @@ func (a *API) RegisterQuerier(queryable storage.Queryable, engine *promql.Engine
 		querier.DummyTargetRetriever{},
 		querier.DummyAlertmanagerRetriever{},
 		func() config.Config { return config.Config{} },
-		map[string]string{}, // TODO: include configuration flags
+		map[string]string{},   // TODO: include configuration flags
+		v1.GlobalURLOptions{}, // TODO: what should this be?
```

Comment:
The GlobalURLOptions looks used only by `/targets` API endpoint, which Cortex doesn't use. I think we can safely remove the TODO comment.
```suggestion
		v1.GlobalURLOptions{},
```